src/sidebar/services/test/groups-test.js

```diff
@@ -253,6 +253,93 @@ describe('groups', function() {
         });
       });
     });
+
+    it('shows service groups when logged in', () => {
```

Comment:
In the LMS context the user will always be logged in, so we don't have to worry too much about what happens if `services` is present and non-empty but the user is not logged in (ie. `services[0]` has no `grantToken` entry). However I would suggest we always respect the `groups` list if `services` is present & non-empty for consistency.